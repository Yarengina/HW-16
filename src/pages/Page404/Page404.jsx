// eslint-disable-next-line no-unused-vars
import * as React from 'react'
import Navigation from '../../components/Navigation/Navigation'

function Error() {
    return (
        <>
            <Navigation />
            <main>
                <h2>Такой страницы здесь нет</h2>
            </main>
        </>
    )
}

export default Error
