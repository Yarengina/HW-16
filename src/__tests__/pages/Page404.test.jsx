// eslint-disable-next-line no-unused-vars
import * as React from 'react'
import { render } from '../utils/renderWithRouter'
import Page404 from '../../pages/Page404/Page404'

describe('<Page404 />', () => {
    it('should render successfully and match snapshot', () => {
        const { container } = render(<Page404 />)
        expect(container).toMatchSnapshot()
    })
})
