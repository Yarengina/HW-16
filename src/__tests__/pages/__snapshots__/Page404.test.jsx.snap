// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page404 /> should render successfully and match snapshot 1`] = `
<div>
  <nav
    class="LinkField"
  >
    <a
      aria-current="page"
      class="Link active"
      data-testid="login-link"
      href="/"
    >
      <span>
        Login
      </span>
    </a>
    <a
      class="Link"
      data-testid="about-link"
      href="/about"
    >
      <span>
        About
      </span>
    </a>
    <a
      class="Link"
      data-testid="order-link"
      href="/order"
    >
      <span>
        Order
      </span>
    </a>
  </nav>
  <main>
    <h2>
      Такой страницы здесь нет
    </h2>
  </main>
</div>
`;
